saos-webapp/src/main/webapp/WEB-INF/static/javascript/modules/formSection.js

```diff
@@ -0,0 +1,197 @@
+/*
+ * Form section
```

Comment:
It's not just any form section. It's a pair of bounded info-form section. The name: infoFormSection.js would be more explanatory.